### PR TITLE
Align Review Committee Rules with Practice

### PR DESCRIPTION
**Pull request by tjablin**

### CODE REVIEW

File: submission_rules.adoc [hunk 1]
```diff
@@ -21,70 +21,91 @@ Unless otherwise stated all rules apply to closed and open.
...
 ### Structure
 
 The review committee consists of the following people:
 
+*   The relevant working group chairs
+*   The relevant power working group chairs
+*   Optionally, the review chair of relevant working group
+*   The executive director
+*   The president of the board
+*   One representative of each submitter
```
Comment (christ1ne) [inline submission_rules.adoc:36]: I think we have multiple representatives from one submitters before. I addition to "One representative of each submitter.", I recommend we allow up to 3 observers from each submitter for providing clarifications in the review process.

File: submission_rules.adoc [hunk 2]
```diff
@@ -21,70 +21,91 @@ Unless otherwise stated all rules apply to closed and open.
...
 
-The review committee should attempt to decide issues through discussion and grudging consensus whenever possible. However, if the review committee and all submitters are unable to reach a grudging consensus, the review committee will vote to decide the issue.
+The review committee should attempt to decide issues through discussion and
+grudging consensus whenever possible. However, if the review committee is unable
+to reach a grudging consensus, the submitters will vote to decide the
+issue. Non-submitters may not vote, except in the case of a tie. In the case of
+a tie, the first non-submitter from the following list will cast the deciding
+vote: the review chair, the chairs of the relevant working group shortest tenure
+first, the chairs of the relevant power working group shortest tenure first, and
+a random number generator.
```
Comment (christ1ne) [inline submission_rules.adoc:71]: how about adding "the executive director and the president of the board are not allowed in the voting" just for completeness?

**Comment by christ1ne** [comment]
@rnaidu02 just FYI.

File: submission_rules.adoc (hunk 2 above)
Comment (tjablin) [reply submission_rules.adoc]: Done.

File: submission_rules.adoc (hunk 1 above)
Comment (tjablin) [reply submission_rules.adoc]: Done.

File: submission_rules.adoc [hunk 3]
```diff
@@ -21,70 +21,91 @@ Unless otherwise stated all rules apply to closed and open.
...
-*   When a permanent host organization exists:
-    **   The executive director
-    **   The chairperson of the board
-*   For v0.5 only, and only for the review process: One representative of each accelerator vendor (as opposed an OEM or system vendor) who submitted and is not otherwise represented above, provide that in the opinion of the majority of the above said representative is a significant contributor to MLPerf through active participation in one or more working groups.
-
-If representatives of a single organization, its parents, subsidiaries, affiliates, or contractors hold multiple positions on this list, only one such representative is eligible to serve on the review committee. 
-
-The review committee is chaired by the relevant results working group chair, e.g. training results working chair during a training submission review. For this reason, the results working groups may never have multiple chairs. 
-
+The review committee is chaired by the relevant results working group chair
```
[the remark this answers is not in the document]
Comment (tjablin) [reply submission_rules.adoc]: Done.

**Comment by christ1ne** [comment]
LGTM

**Review by christ1ne — APPROVED** [review]

**Comment by DilipSequeira** [comment]
Looks great, except I would prefer tiebreaks decided by a sequence such as:

* the non-submitter review chair votes
* the non-submitter WG chairs either all vote, or delegate to the non-submitter power chairs to all vote
* a RNG.

Since the Power chairs' authority should be limited to Power matters, it's not clear to me that length of tenure is necessarily clear, and even if clear, shortest first seems arbitrary.

**Comment by tjablin** [comment]
There are always two WG chairs. If both chairs are non-submitters, there could be a tie. Consequently, I'd prefer the rule added exactly one tie breaker vote. If we have order the non-submitter WG chairs, I thought length of tenure was an obvious total ordering. Choosing the longest tenured chair creates a small incentive to stay WG chair as long as possible, choosing the shortest tenured chair creates a small incentive for turnover. I thought turnover would we more healthy for the organization. How would you feel about cutting the power chair from the list and choosing the order of non-submitter WG chairs randomly?

**Comment by DilipSequeira** [comment]
I'm fine with cutting the Power chair - the relevant WG chair(s) would presumably consult them on relevant matters before voting.

Then I'm also fine with choosing an eligible tie-breaking WG chair randomly; if the WG chairs agree we only need one, and if they disagree the answer will be decided randomly anyway.

**Comment by tjablin** [comment]
Dilip, I have modified the proposal to address your concerns. PTAL.

**Comment by tjablin** [comment]
No objections from Inference WG.

**Review by johntran-nv — APPROVED** [review]

**Pull request merged**